library/cmac.c

```diff
@@ -0,0 +1,869 @@
+/*
+ *  NIST SP800-38B compliant CMAC implementation
```

Comment:
Should use doxygen \file and \brief tags. (These tags are used inconsistently in the library, but is the style it's *supposed* to be).